doc/application_workflow/harbor_registry/harbor_with_fqdn.md

```diff
@@ -0,0 +1,223 @@
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:California
+Locality Name (eg, city) []:San Francisco
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:VMware
+Organizational Unit Name (eg, section) []:vic
+Common Name (e.g. server FQDN or YOUR name) []:<Habor FQDN here!>
+Email Address []:
+
+Please enter the following 'extra' attributes
+to be sent with your certificate request
+A challenge password []:password
+An optional company name []:
+loc@Devbox:~/mycerts$ ls
+ca.crt  ca.key  server.csr  server.key
+```
+Notice above, we used the name server.key and server.csr to make it clear.  This is a deviation from the Harbor documentation.
+
+Sign the server certificate:  (subjectAltName is required.  It is the IP from above.)
+
+```
+loc@Devbox:~/mycerts$ mkdir demoCA
+loc@Devbox:~/mycerts$ cd demoCA
+loc@Devbox:~/mycerts/demoCA$ touch index.txt
+loc@Devbox:~/mycerts/demoCA$ echo '01' > serial
+loc@Devbox:~/mycerts/demoCA$ cd ..
+loc@Devbox:~/mycerts$ echo subjectAltName = IP:HarborVM_IP > extfile.cnf
+
+loc@Devbox:~/mycerts$ openssl ca -in server.csr -out server.crt -cert ca.crt -keyfile ca.key -extfile extfile.cnf -outdir .
+Using configuration from /usr/lib/ssl/openssl.cnf
+Check that the request matches the signature
+Signature ok
+Certificate Details:
+        Serial Number: 1 (0x1)
+        Validity
+            Not Before: Nov 11 23:53:26 2016 GMT
+            Not After : Nov 11 23:53:26 2017 GMT
+        Subject:
+            countryName               = US
+            stateOrProvinceName       = California
+            organizationName          = VMware
+            organizationalUnitName    = vic
+            commonName                = <Habor FQDN here!>
+        X509v3 extensions:
+            X509v3 Subject Alternative Name: 
+                IP Address:<Habor FQDN here!>
+Certificate is to be certified until Nov 11 23:53:26 2017 GMT (365 days)
+Sign the certificate? [y/n]:y
+
+
+1 out of 1 certificate requests certified, commit? [y/n]y
+Write out database with 1 new entries
+Data Base Updated
+loc@Devbox:~/mycerts$ ls
+01.pem  ca.crt  ca.key  demoCA  extfile.cnf  server.crt  server.csr  server.key
+loc@Devbox:~/mycerts$ rm server.csr
+
+```
+
+In the above, we only care about 3 files: ca.crt, server.crt, server.key.  We will use the server cert/key files to update the Harbor server.  We will use the ca.crt to update standard docker and our VCH. 
+<br><br>
+
+## Updating the Harbor VM with the server certificate and key
+
+We now the server.crt file.  Make sure to open file with an editor that does not modify the line ending.  Here, we use Notepad++ on Windows.  Any text editor on Linux or Mac will do.  After opening the file, we see a lot of information, we skip right to where we see ------BEGIN CERTIFICATE------.  We copy the entire certificate into our clipboard.
```

Comment:
we now the server.crt file ?

"After opening the file, we see a lot of information, we skip ..." --> "we see a lot of information; we skip ..."